commit bd0fbbb841d060e2cac2d929baf220474c3f9d7a (refs/changes/61/4461/1)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-06-29T18:14:33+00:00 (1 year, 3 months ago)
    
    TRITON-563 net-agent dumps core due to attempt to change child logger's name

diff --git a/lib/aggr-fsm.js b/lib/aggr-fsm.js
index 04490e4..a874297 100644
--- a/lib/aggr-fsm.js
+++ b/lib/aggr-fsm.js
@@ -86,7 +86,7 @@ function AggrFSM(opts) {
     this.app = opts.app;
     this.log = opts.app.log.child({
         component: 'aggr',
-        name: this.name
+        aggr: this.name
     }, true);
 
     this.pending = {
diff --git a/lib/server-fsm.js b/lib/server-fsm.js
index 9fb09aa..7019b98 100644
--- a/lib/server-fsm.js
+++ b/lib/server-fsm.js
@@ -43,6 +43,7 @@ function ServerFSM(opts) {
     }, true);
 
     this.nics = {};
+    this.aggrs = {};
     this.nictags = {};
 
     mod_mooremachine.FSM.call(this, 'init');
